test-ciphers.sh

```diff
@@ -1,56 +1,66 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
-CONSOLE_URL=$(oc get console.config.openshift.io cluster --template '{{.status.consoleURL}}')
-if [ -z "${CONSOLE_URL}" ]
-then
-  echo "no console_url, are you connected to a cluster?"
-  exit 1
+oc project openshift-console
+
+CONSOLEPOD=$(oc get pods --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep --max-count 1 console)
+CONSOLE_POD_IP=$(oc get pod "${CONSOLEPOD}" --template '{{.status.podIP}}')
+
+echo "pod ${CONSOLEPOD} at ${CONSOLE_POD_IP}"
+
+POD_NAME="test-console-ciphers"
+CONFIGMAP_NAME="${POD_NAME}"
+FILE_NAME="test-ciphers-from-pod.sh"
+FILE_PATH="/scripts"
+
+
+# clean up after last run
+oc delete pod "${POD_NAME}"
+oc delete configmap "${CONFIGMAP_NAME}"
+
+# put the test script in a configmap in the console namespace
+oc create configmap "${CONFIGMAP_NAME}" --from-file="./test/${FILE_NAME}"
+
+# creates a POD to run the test script in the console namespace
+# passes $CONSOLE_POD_IP as $SERVER environment variable
+cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: openshift-console
+  name: "${POD_NAME}"
+spec:
+  restartPolicy: Never
+  containers:
+  - name: "${POD_NAME}"
+    image: quay.io/benjaminapetersen/console-ciphertest:latest
+    command: ["/bin/bash"]
+    args:
+    - ${FILE_PATH}/${FILE_NAME}
+    env:
+    - name: SERVER
+      # :8443
+      value: "${CONSOLE_POD_IP}:8443"
+    volumeMounts:
+    - name: "${CONFIGMAP_NAME}"
+      mountPath: ${FILE_PATH}
+  volumes:
+  - name: "${CONFIGMAP_NAME}"
+    configMap:
+      name: "${CONFIGMAP_NAME}"
+      defaultMode: 0755
+EOF
+
+# wait to ensure the tests run
+sleep 10
+
+# check the logs to see that ciphers were handled as expected
+LOGS=$(oc logs "${POD_NAME}")
+echo "${LOGS}"
```

Comment:
dumping the logs from the pod to make debugging easier if this ever breaks for some reason